types/src/front/membership_invitation.ts

```diff
@@ -10,6 +10,7 @@ export type MembershipInvitationType = {
   status: "pending" | "consumed" | "revoked";
   inviteEmail: string;
   initialRole: ActiveRoleType;
+  createdAt: Date;
```

Comment:
Let's use a number here (epoch) to avoid any surprises as we go through the network

```
createdAt: number
```